### PR TITLE
tests: fix muinstaller tests for 24

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**

### CODE REVIEW

File: tests/lib/muinstaller/mk-classic-rootfs.sh [hunk 1]
```diff
@@ -100,7 +100,13 @@ if [ -f /cdrom/casper/base.squashfs ]; then
...
     BASETAR=ubuntu-base.tar.gz
     # important to use "-q" to avoid journalctl suppressing  log output
-    wget -q -c http://cdimage.ubuntu.com/ubuntu-base/releases/22.04/release/ubuntu-base-22.04.1-base-amd64.tar.gz -O "$BASETAR"
+    release=$(lsb_release -r -s)
+    if [ "$release" = 22.04 ]; then
+        pointrel=.4
+    else
+        pointrel=
+    fi
+    wget -q -c http://cdimage.ubuntu.com/ubuntu-base/releases/"$release"/release/ubuntu-base-"$release""$pointrel"-base-amd64.tar.gz -O "$BASETAR"
```
Comment (bboozzoo) [inline tests/lib/muinstaller/mk-classic-rootfs.sh:109]: too bad there's no current/latest symlink

**Review by bboozzoo — APPROVED** [review]
LGTM

File: tests/nested/manual/muinstaller-core/task.yaml [hunk 2]
```diff
@@ -75,7 +75,11 @@ execute: |
   # Retrieve kernel
   snap download --basename=pc-kernel --channel="$version/edge" pc-kernel
   # Build kernel with initramfs with the compiled snap-bootstrap
-  uc20_build_initramfs_kernel_snap "$PWD/pc-kernel.snap" "$NESTED_ASSETS_DIR"
+  if os.query is-ubuntu-ge 24.04; then
+    uc24_build_initramfs_kernel_snap "$PWD/pc-kernel.snap" "$NESTED_ASSETS_DIR"
```
Comment (andrewphelpsj) [inline tests/nested/manual/muinstaller-core/task.yaml:79]: possibly worth putting this conditional inside of a generic `build_initramfs_kernel_snap` function?

**Review by andrewphelpsj — APPROVED** [review]
Looks good, thanks!

**Pull request closed without merging**